cmd/nmap.go

```diff
@@ -217,6 +217,7 @@ func init() {
 	nmapCmd.Flags().BoolVarP(&skipHTTP, "no-http", "s", false, "Skip trying to connect with HTTP")
 	nmapCmd.Flags().BoolVarP(&skipHTTPS, "no-https", "S", false, "Skip trying to connect with HTTPS")
 	nmapCmd.Flags().IntVarP(&maxThreads, "threads", "t", 4, "Maximum concurrent threads to run")
+	nmapCmd.Flags().BoolVarP(&nmapOnlyOpenPorts, "open","",false,"Nmap ports to filter by. Only uses ports with an open state")
```

Comment:
Should the help here not just say "Only select open ports"?

Reply:
Makes sense to me, I added another commit which changed the wording to the above. If you want to add a single flag letter feel free to change if you merge with master.